app/assets/javascripts/analytics/init.js.erb

```diff
@@ -43,6 +43,7 @@ var analyticsInit = function() {
       'access.service.gov.uk',
       'access.tax.service.gov.uk',
       'accounts.manage-apprenticeships.service.gov.uk',
+      'account.publishing.service.gov.uk',
```

Comment:
The account manager is on www.account.publishing.service.gov.uk

Reply:
Also this won't be sufficient for tracking sign ins, as we'll need to figure out how to get the _ga parameter passed through some redirects

Reply:
Yes. GA automatically applies subdomains based on these strings. So for example, if we just put `gov.uk` in, it would match everything in this list (but we don't, for good reasons).

Reply:
Let's have a separate conversation about redirects then, but the change in this PR still needs doing.